src/xkbcomp/action.c

```diff
@@ -169,16 +171,22 @@ static const LookupEntry fieldStrings[] = {
     { NULL,               0                          }
 };
 
-static bool
```

Comment:
The previous signature is less likely to be misused, so I prefer it. But no hard objection.

Reply:
Agreed, but when I tried to rewrite it to fix the enum pointer cast it somehow felt off and inefficient. Since these 2 functions are only called twice, maybe we could just inline them?

Reply:
Type-safe wrappers seem like a good thing to me. Personally I'd add the bit of boilerplate but up to you :)

Reply:
Let’s do your way then.